### ISSUE TITLE
Kali repository not removed

### ISSUE THREAD

**Issue by MD-AZMAL** [LionSec/katoolin#197]
I removed kali repository its not in the sources.list
but while updating ubuntu it still shows up.
How to remove it completely?

**Comment by scitronboy** [comment]
Did you use katoolin to remove the kali repo?

**Comment by MD-AZMAL** [comment]
yes i added repos first but installation didnt worked so i had to remove it ... but it wasnt removed

**Comment by scitronboy** [comment]
Hmm. Try removing them manually. Type `sudo nano /etc/apt/sources.list`.

**Comment by Georgemwaniki** [comment]
which repositories can i use to update my kali linux
